### PR TITLE
Chore: Split DragLayer out of the CM and make it available across the app

### PR DESCRIPTION
**Pull request by gu-stav**
### What does it do?

This splits the `DragLayer` component away from the CM and makes it available across the app.

### Why is it needed?

It would be nice to have one generic way to render dragged items. The types can be dragged around are specific to each part of the app and are therefore not part of the component anymore.

This will help not to duplicate the drag-layer for review workflows.

### How to test it?

Everything should work as before.

### CODE REVIEW

File: packages/core/admin/admin/src/content-manager/components/DragLayer/index.js [hunk 1]
```diff
@@ -1,85 +1,48 @@
...
+      );
+
+    default:
+      return null;
+  }
+}
 
+export function DragLayer() {
   return (
     <LayoutDndProvider>
```
[the remark this answers is not in the document]
Comment (gu-stav) [reply packages/core/admin/admin/src/content-manager/components/DragLayer/index.js:44]: I have not but you that you say it ... Will have a look.

**Comment by Boegie19** [comment]
Would it not maybe make more since to move it to the design system? so that it is also easier for other plugins to use? + then you can have a standard for drag and drop inside the cms + plugins

**Comment by gu-stav** [comment]
@Boegie19 I'm not against it if there is a need for it, but right now the DS does not have react-dnd as dependency and introducing it there would mean we'd have many questions to answer. If you think it is needed I'd encourage you to open an issue over there, so that we can start a wider discussion :)

File: packages/core/admin/admin/src/content-manager/pages/App/components/CardDragPreview.js
Comment (joshuaellis) [inline packages/core/admin/admin/src/content-manager/pages/App/components/CardDragPreview.js:1]: If this is used across multiple pages does it still make sense to have it under a page?

**Review by joshuaellis — APPROVED** [review]
Looks good 🚀 i've left a question but it's not blocking.

File: packages/core/admin/admin/src/content-manager/pages/App/components/CardDragPreview.js
Comment (gu-stav) [reply packages/core/admin/admin/src/content-manager/pages/App/components/CardDragPreview.js:1]: This is why I moved it into the global app scope :)

Comment (joshuaellis) [reply packages/core/admin/admin/src/content-manager/pages/App/components/CardDragPreview.js:1]: ah, sorry i assumed `App` was a single page

**Pull request merged**